Wbooks/src/app/screens/library/components/bookList/index.js

```diff
@@ -1,27 +1,60 @@
 import React, { Component } from 'react';
-import { FlatList } from 'react-native';
+import { FlatList, View } from 'react-native';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
 
 import Book from '../book/index';
-import mockData from '../../../../mocklist.json';
+import BookActions from '../../../../../redux/book/actions';
+import bookModel from '../../../../proptypes/bookModel';
 
 import styles from './styles';
 
 class BookListContainer extends Component {
-  renderItem = ({ item }) => <Book book={item} navigation={this.props.navigation} />;
+  handlePress = item => {
+    const { navigation, loadBook } = this.props;
+    loadBook(item);
+    navigation.navigate('BookDetail', { navigation });
```

Comment:
Does `bookDetail` use the navigation param? Oh, and pass the route name constant instead of a string.